build/scripts/build.ps1

```diff
@@ -38,6 +38,7 @@ param (
     [switch]$testDesktop = $false,
     [switch]$testCoreClr = $false,
     [switch]$testIOperation = $false,
+    [switch]$noRedirect = $false,
```

Comment:
Should prefer "skip" vs "no" here as it's the predominant pattern in other repositories. I realize there is already "noanalyzers" but that was done without my review. It should be changed as well at some point.

Reply:
Should put this with the rest of the configuration values above.